gui-wm/labwc/labwc-0.7.2.ebuild

```diff
@@ -18,18 +18,21 @@ fi
 
 LICENSE="GPL-2"
 SLOT="0"
-IUSE="+X"
+IUSE="+nls +svg +X"
 
 RDEPEND="
 	dev-libs/glib:2
 	dev-libs/libinput
 	dev-libs/libxml2:2
 	>=gui-libs/wlroots-0.17.0:=[X?]
+	media-libs/libpng
 	x11-libs/cairo[X?]
 	x11-libs/libdrm:=
 	x11-libs/libxkbcommon:=[X?]
 	x11-libs/pango[X?]
 	x11-libs/pixman
+	nls? ( sys-devel/gettext )
+	svg? ( gnome-base/librsvg:= )
```

Comment:
How is svg used?

Reply:
That would be your doing in the 0.7.1 ebuild :p https://github.com/bsd-ac/wayland-desktop/commit/6088c3261569d15cc9c48431b227d3054ef5dd82

Removing 0.7.0 and adding 0.7.2 made git think I was renaming the file instead of the two separate actions (I should have probably made two commits instead of just one). 

What I actually did was remove 0.7.0, copy 0.7.1 -> 0.7.2, read through the ebuild steps (which looked good), took at look through the 0.7.2 changelog and meson.build file (to see if there were any dependency changes) then built the new version. I added the pkg_postinst/pkg_postrm functions because portage picked up on the svg's being installed without updating the cache. Lastly I actually ran the build and it seemed to work properly :)

To answer the question directly, svg is used because of the svgs contained within build process. Particularly, the line in meson.build has `svg = dependency('librsvg-2.0', version: '>=2.46', required: false)`

Reply:
Well, seems like I was wrong with that ebuild T_T
Can you also add options in emesonargs with something like `$(meson_feature svg)` and same for nls.

Reply:
Done :)